proposals/NNNN-Change_ATF_test_reports_folder_structure.md

```diff
@@ -0,0 +1,45 @@
+# Change ATF test reports folder structure
+
+* Proposal: [SDL-NNNN](NNNN-Change_ATF_test_reports_folder_structure.md)
+* Author: [Irina Getmanets](https://github.com/GetmanetsIrina)
+* Status: **Awaiting review**
+* Impacted Platforms: ATF
+
+## Introduction
+
+Main output of ATF is test reports.
+ATF provide such reports:
+* Console logs - name of test cases with execution status FAILED/PASSED.
+* SDL logs - SDL log received by ATF via telnet logger.
+* Transport logs -  all received and sent data from/to SDL. Transport log can have default view - messages on protocol level (except binaries) and full view - log will be expanded with packages: streamings, service messages, heartbeat, json files.   
+* Detailed report - all expected and sent massages related to test cases with expected and actual result.Also contains info about test cases: name, time, sequence, status, duration.
+
+This proposal is about creating clear and useful structure of test scripts reports.
+
+## Motivation
+
+Create more convenient reports structure, less nested. Multiple running of test script should not override old reports. Structure of reports should be clear for easy searching certain report.
```

Comment:
Please update second sentence to "Running multiple test scripts should not override old reports."

Please update third sentence to "Structure of reports should be clear, so you can easily search for a certain report."

Reply:
@theresalech, updated, please find changes in commit 397541b6fb295b86ba1b3bf317b0a5035ff009cd.